grammar_test/literals.txt

```diff
@@ -288,37 +288,10 @@ percent q string with unbalanced delimiters
 ===========================================
 
 %q=a=
-%q!a!
-%q@a@
-%q$a$
-%q%a%
-%q^a^
-%q&a&
-%q*a*
-%q-a-
-%q+a+
-%q/a/
-%q|a|
-%q?a?
-%q\a\
-%q.a.
-%q,a,
-%q:a:
-%q;a;
-%q'a'
-%q"a"
-%q)a)
-%q]a]
-%q}a}
-%q>a>
-%q`a`
-%q~a~
-%q_a_
-%q#a#
```

Comment:
Any chance you could restore these? I’m concerned about the `%q#a#` one in particular.